addon/services/intl.js

```diff
@@ -206,7 +208,17 @@ export default Service.extend(Evented, {
     if (Array.isArray(localeName)) {
       return localeName.map(normalizeLocale);
     }
-  }
+  },
+
+  /** @private **/
+  updateDocumentLanguage: on('localeChanged', function() {
```

Comment:
why do you use the `Evented` system for this instead of calling `updateDocumentLanguage()` directly from the setter?

Reply:
Because it was already there and (in theory) usable. Assuming that `localeChanged` was implemented correctly in the first place, this would have been a "minimally invasive" addition. 😄 

Doing it this way also allows us to potentially add further listeners, without having to explicitly list them all.

But on second thought, I think you are right. The `Evented` thing is foreseen to be deprecated at some point (AFAIK) and the order the listeners would wbe called in is not explicitly defined.

I'll refactor it.